### PR TITLE
Des 137 make the image profile editable

### PR DESCRIPTION
**Pull request by cristianarce89**

### CODE REVIEW

File: .env
Comment (Daniielmayo) [inline .env:1]: change the environment variable, don't leave it on localhost

File: src/auth/registerPageNavbar/ResgisterPageNavbar.jsx [hunk 1]
```diff
@@ -138,7 +142,7 @@ export const RegisterPageNavbar = () => {
             required
             placeholder=" "
             label={'Correo electrónico'}
-             isRequire={true}
+            isRequire={true}
```
Comment (Daniielmayo) [inline src/auth/registerPageNavbar/ResgisterPageNavbar.jsx:145]: The telephone number should not be required in the registration, it must be false

File: src/components/organims/Profile/Profile.css [hunk 2]
```diff
@@ -38,6 +38,113 @@
...
+  border: none;
+  cursor: pointer;
+}
+
+.file-input-label {
+  display: inline-block;
+  line-height: 1.5;
+}
+
+/* .modalEditProfile-circleWrapper {
```
Comment (Daniielmayo) [inline src/components/organims/Profile/Profile.css:102]: These comments are necessary, if not, please delete them.

File: .env [hunk 3]
```diff
@@ -1,2 +1,2 @@
-REACT_APP_API_URL=https://regresemos-cms.herokuapp.com/api
-# REACT_APP_API_URL=http://localhost:8080/api
+# REACT_APP_API_URL=https://regresemos-cms.herokuapp.com/api
```
Comment (Daniielmayo) [inline .env:1]: don't leave localhost

**Review by Daniielmayo — CHANGES_REQUESTED** [review]

**Review by Daniielmayo — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]

**Pull request merged**